addons/udes_stock/models/stock_picking_batch.py

```diff
@@ -474,6 +474,14 @@ def _create_batch(self, user_id, picking_type_id, picking_priorities=None, picki
         if not picking:
             return None
 
+        picking_type = picking.mapped('picking_type_id')
+        reserve_pallet_per_picking = picking_type.filtered('u_reserve_pallet_per_picking')
+        if reserve_pallet_per_picking:
+            max_reservable_pallets = picking_type.mapped('u_max_reservable_pallets')[0]
```

Comment:
We should only have one picking type, so no need for mapped or [0]. We could add ensure_one() after line 477 to enforce this.